express/migrations/migrations/20190128233134-create-user.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      lastName: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
```

Comment:
By convention, we define the name of the table in lowercase and plural and the fields in snake_case, but in model we define the fields in camelCase. So, you must modify the model to this => `lastName: { type: DataTypes.STRING, allowNull: false, field: 'last_name' }`.